src/editor/Gui.cpp

```diff
@@ -410,6 +417,30 @@ void Gui::draw3dContextMenus() {
 					command->execute();
 					app->pushUndoCommand(command);
 				}
+				if (ImGui::MenuItem("Export BSP model", 0, false, !app->isLoading)) {
+					if (app->pickInfo.modelIdx)
+					{
+						Bsp* tmpMap = new Bsp(map->path);
+						tmpMap->modelCount = 1;

```

Comment:
This doesn't need to be set if calling `removed_unused_model_structures` after deleting all entities.